datalad/distributed/ora_remote.py

```diff
@@ -1176,7 +1176,7 @@ def whereis(self, key):
                    self.annex.dirhash(key) + "/" + key + "/" + key
 
         dsobj_dir, archive_path, key_path = self._get_obj_location(key)
-        return str(key_path) if self._local_io() \
+        return Path(dsobj_dir / str(key_path)) if self._local_io() \
```

Comment:
There was a reason for the `str()` call on `key_path`. ;-)

This is ultimately written to stdout which then is read by git-annex. Meaning: Returning a `Path` object isn't useful here, as `annexremote` needs to convert it to string anyway. Plus: `key_path` should already be a `Path` if I'm not mistaken.

Reply:
In short I think `str(dsobj_dir / key_path)` it should be.

Reply:
Sorry, this is all new to me. Would it then rather be 
```
str(Path(dsobj_dir / key_path))
```
?

Reply:
thx! force pushed your suggestion.